### DISCUSSION TITLE
how to set camera positions

### DISCUSSION THREAD

**Discussion by chen-jialan** [superstar54/beautiful-atoms-docs#23] [Q&A]
When I set viewport, and the camera was on the top. I don't know how to focus the render in the part of atoms. Could author tell me how to set parameters to realize this function.
Thanks

**Comment by superstar54** [comment]
Using Python, you can set the `center` when rendering the image.
```python
# set the camera to focus on the second atom.
mol.get_image(viewport = [0, 0, 1], center=mol.positions[2], output = "look_at_2th_atom.png")
```

Using GUI, you can rotate the camera to a particular position.

https://github.com/beautiful-atoms/beautiful-atoms-docs/assets/11457659/019e371a-567b-4a44-8dd5-884174f440fe

**Reply by chen-jialan** [reply]
thanks, I use this way and prefect!
